test/withStylesWithHooks_test.jsx

```diff
@@ -0,0 +1,622 @@
+/* eslint-disable no-unused-vars */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { expect } from 'chai';
+import { mount, shallow } from 'enzyme';
+import sinon from 'sinon-sandbox';
+
+import DirectionProvider, { DIRECTIONS } from 'react-with-direction/dist/DirectionProvider';
+import WithStylesContext from '../src/WithStylesContext';
+import ThemedStyleSheet from '../src/ThemedStyleSheet';
+import {
+  withStylesWithHooks as withStyles,
+  withStylesPropTypes,
+} from '../src/withStylesWithHooks';
+
+describe('withStyles (react >= 16.8)', () => {
```

Comment:
you may need something like `ifReact` from `enzyme-adapter-react-helper`, to skip this test in react < 16.8

Reply:
ping